Documentation/VMware.md

```diff
@@ -23,7 +23,7 @@ Creating a SerenityOS virtual machine is similar to any other virtual machine. T
 4. Choose any size for the hard disk. This disk will later be removed and replaced with the converted GRUB image from the previous stage.
 5. Select **Finish** to finalize creation of the virtual machine.
 6. Select the newly created virtual machine and click **Edit virtual machine settings**.
-7. Serenity requires at minimum 32 MB of memory. Set **Memory for this virtual machine** equal to or above 32 MB. The currently recommended size is 256 MB. Please note that Serenity is currently a 32-bit system, so anything above the ~3.5 GB mark will not be recognized.
+7. Serenity requires at minimum 32 MB of memory. Set **Memory for this virtual machine** equal to or above 32 MB. The currently recommended size is 256 MB.
```

Comment:
32 MiB? That's not going to work at all :D. 

Can you bump the minimum to 512 MiB and the recommended to 1GiB?

Reply:
Thanks, Andrew! I'll push that change up now.

Reply:
Done! Is it preferable to collapse both changes into a single commit?

Reply:
For this tiny a change, yeah. Let's make it one commit. if they affected different sections of the docs, or different md files it might be reasonable to update them independently

Reply:
All done. Let me know if this looks fine now! First commit on SerenityOS, so if I'm missing any small details I apologize and I'm happy to tweak things so they fit style/contribution guidelines.